frameworks/sveltekit.md

```diff
@@ -13,7 +19,55 @@ For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Sv
 :::
 
 ::: tip
-You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs/modules#$service-worker) to disable it on your application.
+If you're using `0.1.*` version of `SvelteKitPWA`, you should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs/service-workers) to disable it on your application.
+:::
+
+## Generate Custom Service Worker
+
+From version `0.2.0`, `SvelteKitPWA` plugin will delegate your custom service worker build to `SvelteKit`, and so, you can use `service-worker` as the name of your service worker file (VanillaJS or TypeScript).
```

Comment:
Is this required or optional? If required then we should change as below:
```suggestion
From version `0.2.0`, `SvelteKitPWA` plugin will delegate your custom service worker build to `SvelteKit`, and so, you must use `service-worker` as the name of your service worker file (VanillaJS or TypeScript).
```

Reply:
uhmm: we can create `src/service-worker.js` (kit will detect it) or `src/service-worker.ts` (we need to configure in bot, kit and pwa options)

On previous versions, we cannot use `src/service-worker.js` to avoid kit to generate the corresponding bundle.

Reply:
If using `src/service-worker.js`, pwa plugin will configure it by default (if missing ofc):
- https://github.com/vite-pwa/sveltekit/pull/32/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R32
- https://github.com/vite-pwa/sveltekit/pull/32/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R35